docs/cli.md

```diff
@@ -0,0 +1,658 @@
+
+<table>
+  <tr>
+  <td markdown="1" >
+
+  ```console
+  $ ls
+  example.txt main.cpp stuff
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli003.png" class="invert-colors-in-dark-mode" alt="ls cli example" width="470px" />
+  </td>
+  </tr>
+</table>
+
+### `pwd`
+`pwd` prints the path of the present working directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ pwd
+  /Users/ohjun/Desktop/project
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli004.png" class="invert-colors-in-dark-mode" alt="ls cli example" width="470px" />
+  </td>
+  </tr>
+</table>
+
+
+### `mkdir`
+`mkdir` creates a directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ mkdir myfolder
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli012.png" class="invert-colors-in-dark-mode" alt="ls cli example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `touch`
+`touch` creates an empty file.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ ls
+  example.txt main.cpp stuff stuff2
+  $ touch euchre.cpp
+  $ ls
+  example.txt main.cpp stuff stuff2 euchre.cpp 
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli006.png" class="invert-colors-in-dark-mode" alt="ls cli example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `rm`
+`rm` removes (deletes) a file. `rm -rf` removes a directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ rm euchre.cpp
+  $ rm -r stuff2/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli013.png" class="invert-colors-in-dark-mode" alt="ls cli example" width="470px" />
+  </td>
+  </tr>
+</table>
+
+### `cd`
+`cd` changes directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ cd stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli014.png" class="invert-colors-in-dark-mode" alt="ls cli example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `mv`
+`mv` moves a file or directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ mv main.cpp stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli009.png" class="invert-colors-in-dark-mode" alt="ls cli example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+Note the `/` at the end of `stuff/`. While not necessary, it is convention to end directory paths with `/`.
+
+<video controls="controls" class="invert-colors-in-dark-mode" width="768px" name="mv gui example">
+  <source src="images/cli_vid001.mov">
+</video>
+
+`mv` is also used to rename a file or directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ mv example.txt new_name.txt
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli015.png" class="invert-colors-in-dark-mode" alt="ls cli example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `cp`
+`cp` copies a file.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ cp new_name.txt stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli016.png" class="invert-colors-in-dark-mode" alt="ls cli example"/>
+  </td>
+  </tr>
+</table>
+
+### `open` / `wslview`
```

Comment:
Do you need to install wslview or is it installed by default?

Reply:
Oh it seems you need to install wslu ([wsl utilities](https://wslutiliti.es/wslu/install.html)) to use it. Based on this [StackExchange post](https://superuser.com/questions/1160419/how-can-i-open-a-file-from-wsl-with-the-default-application), wslu comes pre-installed on some Ubuntu versions, but I am honestly not too sure. It seems a more default-friendly option could be `cmd.exe /C start <file>`. Which do you think we should use?

Reply:
It is straight forward to just `sudo apt install wslu`?

Reply:
Yes, just checked in on a Windows machine running WSL.